File: LYLAcademy-SPA/src/app/models/Participant.ts
import { Calendar } from './Calendar';
import { Student } from './Student';

export class Participant {
    participantId:number;
    studentId:number;
    amount:number;
    amountPaid:number;
    amountRemain:number;
    calendarId:number;
    createDate: Date;
    updateDate: Date;
    student: Student;
    calendar:Calendar;   
    isDelete:number;
}

File: LYLAcademy.API/LYLAcademy.API/Models/Participant.cs
using System;
using System.ComponentModel.DataAnnotations.Schema;

namespace LYLAcademy.API.Models
{
    public class Participant
    {
        public Participant()
        {
            Student = new Student();
        }
        public int ParticipantId { get; set; }
        [ForeignKey("Student")]
        public int StudentId { get; set; }
        public Student Student { get; set; }

        public decimal Amount { get; set; }
        public decimal AmountPaid { get; set; }
        public decimal AmountRemain { get; set; }
        public int CalendarId { get; set; }
        public int IsDelete { get; set; }
        public DateTime CreateDate { get; set; }
        public DateTime UpdateDate { get; set; }



    }
}
